src/github.ts

```diff
@@ -10,28 +38,35 @@ export class Notification {
     payload: any;
     name: string;
     externalID: any;
-    detailsUrl: string;
+    detailsUrl: string = "";
     title: string;
     text: string;
     summary: string;
     count: number;
     conclusion: Conclusion;
+    notificationJobImage: string = "deis/brigade-github-check-run:latest";
```

Comment:
We should update this to `brigadecore/brigade-github-check-run` for the image name.  For tag, I'd vote for the latest tagged release, which is `v0.1.0` currently.  However, we'll have to remember to bump this tag each time a new release is out...